Buildings/Fluid/AirFilters/Empirical.mo

```diff
@@ -0,0 +1,315 @@
+within Buildings.Fluid.AirFilters;
+model Empirical "Empirical air filter model"
+  replaceable package Medium =
+    Buildings.Media.Air(extraPropertiesNames={"CO2"})
+    "Air";
+  parameter Real mCon_nominal(
+    final unit = "kg")
+    "Maximum mass of the contaminant can be captured by the filter";
+  parameter Real epsFun[:]
+    "Filter efficiency curve"
+    annotation (Dialog(group="Efficiency"));
+  parameter Real b(
+    final min = 1 + 1E-3)
```

Comment:
why should we set the `min=1 + 1E-3`, rather than `min=1`? I see in `Buildings.Fluid.AirFilters.BaseClasses.FlowCoefficientCorrection` it says that the `b` has to be greater than 1. Should we have implement a warning?